gfx/libresprite/build

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+
+sed -i '/pthread_detach/d' src/base/thread.cpp
+
+git submodule update --init --recursive
+
+export DESTDIR="$1"
+
+mkdir build && cd build
+cmake \
+  -DUSE_SHARED_CURL=ON \
+  -DUSE_SHARED_ZLIB=ON \
+  -DUSE_SHARED_JPEGLIB=ON \
+  -DUSE_SHARED_LIBPNG=ON \
+  -DUSE_SHARED_LIBWEBP=ON \
+  -DUSE_SHARED_PIXMAN=ON \
+  -DUSE_SHARED_FREETYPE=ON \
+  -DUSE_SHARED_GIFLIB=ON \
+  -DUSE_SHARED_GTEST=ON \
+  -G Ninja ..
+
+ninja -C ./ libresprite
```

Comment:
Please, follow cmake [style-guide](https://k1ss.org/wiki/kiss/style-guide#0800), it works for me.